conf/messages

```diff
@@ -464,6 +464,11 @@ propertyRepresentation.manageAgent.options.unassignFromSomeProperties=Unassign f
 propertyRepresentation.manageAgent.options.assignToYourProperty=Assign to your property
 propertyRepresentation.manageAgent.options.removeFromYourAccount=Remove from your account
 
+propertyRepresentation.manageAgent.assignToAllProperties.question=Are you sure you want to add {0} to all your properties?
```

Comment:
should `add` be `assign` ?

Reply:
I thought the same thing. The text came from the prototype so I don't want to change it without being told to. I will check with aunty Karen.

Reply:
rule number 1: never trust the prototype :)

Reply:
waiting on a response from Karen on this one